.travis.yml

```diff
@@ -14,6 +14,8 @@ env:
     - NPROC_MAX=8 BUILDTEST_MCU_GROUP=cortex_m4
 
 before_install:
+    - source ./dist/tools/pr_check/check_labels.sh
+    - check_gh_label "Ready for CI build" || exit 1
```

Comment:
> Are these lines run in the same environment? I would have imagined Travis launching a new shell instance for each line like make does.

Yes, they are, otherwise it wouldn't work. Will fix for `static-tests` however